README.md

```diff
@@ -3,49 +3,75 @@
+- Get a token from your NowSecure platform instance. More information on this can be found in the [NowSecure Support Portal](https://support.nowsecure.com/hc/en-us/articles/7499657262093-Creating-a-NowSecure-Platform-API-Bearer-Token).
 - Identify the ID of the group in NowSecure Platform that you want your assessment to be included in. More information on this can be found in the
   [NowSecure Support Portal](https://support.nowsecure.com/hc/en-us/articles/38057956447757-Retrieve-Reference-and-ID-Numbers-for-API-Use-Task-ID-Group-App-and-Assessment-Ref).
-  (Note: Authentication is required to access this page)
-- Add a CI/CD variable to your project named, `NS_TOKEN` and add the token created above. As this is a credential, be sure to check the box labeled `Keep this value secret`.
-- Add the following include entry to your project's CI/CD configuration and set your input values
+- Add an [Azure DevOps Secret Variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash#secret-variable-in-the-ui) to your project named, `NS_TOKEN`. Set this to the value of the token created above. 
+- Add the following include entry to your project's CI/CD configuration and set your input values:
 
   ```yaml
   - task: nowsecure-azure-ci-extension@<tag>
     inputs:
+      # Required Inputs    
       group: "<group-ref>"
       token: $(NS_TOKEN)
       binary_file: "<path-to-binary>"
+      # Recommended Optional Inputs
+      minimum_score: 70
+      analysis_type: static
+      artifact_dir: '$(build.artifactStagingDirectory)/NowSecureArtifacts'
   ```
 
-  - `<tag>` is the release tag you want to use
+  Where:
 
-  - `<group-ref>` is uuid of the group that will be used to trigger assessments.
-    Information on how to get the group reference can be found in the [NowSecure Support Portal](https://support.nowsecure.com/hc/en-us/articles/38057956447757-Retrieve-Reference-and-ID-Numbers-for-API-Use-Task-ID-Group-App-and-Assessment-Ref).
-  - `<path-to-binary>` is the filepath for the ipa / apk that is to be uploaded to run an assessments against. Ideally this will be an artifact of some previous build step in a pipeline.
-  - `$NS_TOKEN` is the token used to communicate with the NowSecure API. This token should be an
-    [Azure Devops Secret Variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash#secret-variable-in-the-ui).
-    Information on how to create a token can be found in the [NowSecure Support Portal](http://support.nowsecure.com/).
+    - `task` is specifying the task to run.  In the example, `<tag>` is the release tag of the `nowsecure-azure-ci-extention` that you want to use.  At time, NowSecure recommends setting this to `0.1`
```

Comment:
Sadly ADO only supports major version or full semver. So we need to recommend either `@0` or `@0.1.0`